aea/cli/common.py

```diff
@@ -496,3 +507,77 @@ def _find_item_locally(ctx, item_type, item_public_id) -> Path:
         sys.exit(1)
 
     return package_path
+
+
+def _check_package_fingerprints(
+    package_configuration: PackageConfiguration, package_directory: Path
+):
+    """Given a package configuration and a package directory, check whether
+    the hash of the files in the package directory match the one in the package configurations."""
+    hasher = IPFSHashOnly()
+    expected_fingerprints = package_configuration.fingerprint
+
+    # compute fingerprints
+    actual_fingerprints = {}  # type: Dict[str, str]
+    # find all valid files of the package
+    all_files = [
+        x
+        for x in package_directory.glob("**/*")
+        if x.is_file()
+        and not x.match("*__pycache__/*")
+        and not x.match(
+            "*.yaml"
+        )  # TODO this should only match "protocol.yaml", "connection.yaml" etc.
+    ]
```

Comment:
this finds all files in a directory, excluding:
- the ones in `__pycache__`
- all the `*.yml` files (including the package config files)